sdk/storage/Azure.Storage.Files.DataLake/src/Models/ChangeAccessControlListPartialResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// ChangeAccessControlListPartialResult contains partial counts of operations that change Access Control Lists recursively.
+    /// Additionally it exposes path entries that failed to update while these operations progress.
+    /// See <see cref="DataLakePathClient.SetAccessControlListRecursive(System.Collections.Generic.IList{PathAccessControlItem}, int?, System.IProgress{ChangeAccessControlListPartialResult}, System.Threading.CancellationToken)"/>
```

Comment:
Adding the see crefs to the APIs that use the model would be a new convention.  Is this something you'd like us to do going forward?

Reply:
I'm going to remove it. It's easy to forget to update this if we add more APIs referencing that.